### DISCUSSION TITLE
Upgraded to 1.0.4 from 0.0.8 - all entities are gone due to renaming

### DISCUSSION THREAD

**Discussion by GrimmiMeloni** [OStrama/weishaupt_modbus#73] [Q&A]
I just realized I was still stuck with my own fork based off of 0.0.8 that I had created a few weeks ago to contribute the wbb 8 kennfeld.
So I made the jump to the official repo and installed the current 1.0.4. By doing so, all my entities have now new names. Argh. 

I see, there are two prefixing checkboxes in the configuration dialog that seem to be related. Is there any combination in that dialgue to select to get back the old names? 

Or is there any way for me to migrate my existing setup, or do I really have to toss all? I am not in love with the previous entity names, so happy to stick with the new notation. But having to redo all the dashboards feels kinda tedious.

I saw the mentioning of the rename tool, but I cannot run it on the console via python3 due to a missing requests package? 

cc @OStrama @MadOne

**Comment by OStrama** [comment]
Did you try to delete the integration, install it again and restart HA?
I tried to di a rename of the entities to the new names, but this works not stable due to issues in the recorder.
There are two possibilities:
- use the rename tool
- Rename the entities back to the old names manually. Just remove the device name part..
-

**Reply by GrimmiMeloni** [reply]
I'll probably just bite the bullet. Manually renaming to old does not solve that many things for me, and also creates manual effort. I'd rather stick with the name scheme then.

My question around those prefix checkboxes in the configuration dialog got lost in all that I think. I'd would be great if you could   explain what these do and more importantly if I have to pick a specific combination, to ensure I don't have to rename again in the future.

Regarding the renaming tool - I think this needs a small readme. At least to me it is not clear how to use it. Simply starting it does not work for me. I probably am doing it wrong somehow?

```python
$ cd /Users/mhess/development/github.com/OStrama/weishaupt_modbus/rename_tool
$ python3 entity_rename.py
Traceback (most recent call last):
  File "/Users/mhess/development/github.com/OStrama/weishaupt_modbus/rename_tool/entity_rename.py", line 6, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

**Reply by MadOne** [reply]
Renaming dashboard should be fairly easy. Go to full yaml mode and search & replace

**Reply by GrimmiMeloni** [reply]
Good point. I always forget about the yaml views in the UI. Thanks. Fixed.
